src/Form/VehicleType.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Form;
+
+use App\Entity\Vehicle;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Form\Extension\Core\Type\TextareaType;
+
+class VehicleType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder
+            ->add('name')
+            ->add('description', TextareaType::class)
+            ->add('type')
+            ->add('fiscalPower')
+            ->add('actualPower')
+            ->add('tankCapacityCNG')
+            ->add('consumptionCNG')
+            ->add('tankCapacityFuel')
+            ->add('consumptionFuel')
+            ->add('autonomy')
+            ->add('rearVolume')
+            ->add('photo')
```

Comment:
You shouldn't rely on the type guesser and specify the Type for all your form fields